### PR TITLE
Add new glossary entries

### PR DESCRIPTION
**Pull request by nshlapo**
Adding glossary entries chosen during the August Marketing and Solutions planning session.

- Digital Twin
- Single Synthetic Environment
- Multi-Agent Systems
- Business Intelligence
- Ego Networks
- Graph Representation Learning
- Knowledge Graph Machine Learning
- Data Mesh
- Parameters
- Optimization Methods
- Robotic Process Automation
- Content Management System
- Customer Relationship Management System
- Applicant Tracking System
- Enterprise Resource Planning
- Project Management Software
- Robustness

### CODE REVIEW

File: resources/glossary/applicant-tracking-system.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Applicant Tracking System
+description: "Applicant tracking systems help employers manage recruitment and hiring."
+slug: applicant-tracking-system
+tags: ["Databases", "Artificial Intelligence"]
```
Comment (ghabs) [inline resources/glossary/applicant-tracking-system.mdx:5]: add a tag for "SaaS" or "Enterprise Software"

File: resources/glossary/content-management-system.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Content Management System
+description: "Content management systems allow you to build and manage websites."
+slug: content-management-system
+tags: ["Databases", "Software Development"]
```
Comment (ghabs) [inline resources/glossary/content-management-system.mdx:5]: same, enterprise software, or some tag that unifies ATS and CMS and CRM

File: resources/glossary/single-synthetic-environment.mdx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Single Synthetic Environment
+description: "Digital twins are a detailed, simulated analogue to a real-world system"
```
Comment (ghabs) [inline resources/glossary/single-synthetic-environment.mdx:3]: description is wrong

File: resources/glossary/single-synthetic-environment.mdx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+slug: single-synthetic-environment
+tags: ["Simulation Modeling", "Databases", "Data Science"]
```
Comment (nonparibus) [inline resources/glossary/single-synthetic-environment.mdx:5]: Not sure about the _databases_ tag here - would remove it.

More broadly I'm a bit unsure about the _databases_ tag in general. It seems to me that (this aside) the other places we utilize it we could well replace it with "Integrable Systems" or "Integrations", to indicate that these definitions refer to types of software/platforms/systems that can be linked up to HASH.

@nshlapo @ghabs

Comment (ghabs) [reply resources/glossary/single-synthetic-environment.mdx:5]: Agree databases shouldn't apply here. If we think of the tags as being focused around how these products/concepts work, integrations is a good  change

**Review by ghabs — APPROVED** [review]

**Pull request merged**